packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js

```diff
@@ -104,11 +104,15 @@ const withLocalNPMRepo = (func) => {
             // packages to it. This is safe to do – Verdaccio does not forward these
             // the public NPM repo.
             console.log('Publishing packages to the local NPM repository')
-            sh.exec('npm run lerna -- publish from-package --yes --concurrency 1 --loglevel warn', {
```

Comment:
Please ignore any changes to the files in `pwa-kit-create-app`. They're just for my own testing purpose. I plan to undo the changes.